app.js

```diff
@@ -0,0 +1,96 @@
+const { ValidateName, ValidateEmail, ValidateUsername } = require("./src/validation/validation");
+const express = require('express');
+const app = express();
+const bcrypt = require('bcrypt');// for hashing and also comparing(not used yet).
+app.use(express.urlencoded({extended:false}));//used in post method for data of req.
+const mongoose = require("mongoose");
+app.use(express.static('public'))
+
+
+//mongo connectivity...
+mongoose.connect('mongodb://localhost:27017/variant', {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+});
+var db = mongoose.connection;
+
+db.on('error', () => console.log("Error in connecting to database"));
+db.once('open', () => console.log("connected to Database"));
+
+// creation of mongoose schema..
+const Schema = mongoose.Schema;
```

Comment:
Maintain a separate folder, files for the schema. 
Just a good practice of maintaining the code structure.